content/300-guides/200-deployment/110-deployment-guides/500-deploying-to-netlify.mdx

```diff
@@ -215,7 +192,7 @@ Create a variable with the key `DATABASE_URL` and set the value with your Postgr
 
 > **Note:** It's considered best practice to keep secrets out of your codebase. If you open up the `prisma/schema.prisma` file, you should see `env("DATABASE_URL")` in the _datasource_ block. By setting an environment variable you keep secrets out of the codebase.
```

Comment:
That's a bit of a weird note here. Maybe mention that at step 4 instead, where they put it into their `.env` files instead of `schema.prisma` directly.

Reply:
This note is now deleted.

Reply:
The section about the DATABASE_URL local variable is now based on using .env.